docs/scripts/api/buildComponentsDocumentation.ts

```diff
@@ -295,7 +295,14 @@ const buildComponentDocumentation = async (options: {
 
   const componentApi: {
     componentDescription: string;
-    propDescriptions: { [key: string]: string | undefined };
+    propDescriptions: {
+      [key: string]: {
+        description: string;
+        requiresRef?: string;
```

Comment:
```suggestion
        requiresRef?: boolean;
```

Reply:
Would it make more sense to reuse the core interface?

```ts
import type { ReactApi as CoreReactApi } from '@mui/monorepo/packages/api-docs-builder/ApiBuilders/ComponentApiBuilder';

// ...

const componentApi: CoreReactApi['translations'] = {
```

Reply:
Yes, would perfectly make sens, I had an import issue, but probably related to another modification I was working on 👍